conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -8,21 +8,16 @@
 FEEDSTOCK_ROOT=$(cd "$(dirname "$0")/.."; pwd;)
 RECIPE_ROOT=$FEEDSTOCK_ROOT/{{ recipe_dir }}
 
-docker info
```

Comment:
Why drop this? Knowing the version and configuration of `docker` is incredibly helpful.